src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -456,4 +466,8 @@ export default class CreateSpecialResolution extends Mixins(DateMixin) {
 .tool-tip-text{
    border-bottom: 1px dashed $gray6;
 }
+
+::v-deep .invalid-section label.v-label.error--text ,.invalid-section label{
```

Comment:
This could use a comment indicating why it's needed (what are you targeting exactly?)

Also please fix the whitespace around the comma and before the curly brace.